.github/ISSUE_TEMPLATE/crash.yml

```diff
@@ -0,0 +1,73 @@
+name: Crash report
+description: A hard crash of the interpreter, possibly with a core dump
+labels: ["type-crash"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+          This form is for hard crashes of the Python interpreter, segmentation faults, failed C-level assertions, and similar.
+          Unexpected exceptions raised from Python functions in the standard library count as bugs rather than crashes.
+
+          The CPython interpreter is written in a different programming language, C.
+          A "CPython crash" is when Python itself fails, leading to a traceback in the C stack.
```

Comment:
Let's tidy up the newlines here, either single or double newlines between (single-sentence) paragraphs, and optionally merge paras if you like.

![Screenshot_20230813_232408_Chrome](https://github.com/python/cpython/assets/1324225/dd8907d1-2cbe-469b-9e46-70ed4a17f47e)

And let's be consistent in the other forms, like the featurw request.

Reply:
Ahh, thanks for testing on mobile! I'd only tested on my laptop, where the line breaks looked fine.